packages/bookshop/srv/external.cds

```diff
@@ -0,0 +1,76 @@
+using { API_BUSINESS_PARTNER as external } from './external/API_BUSINESS_PARTNER.csn';
+
+/**
+ * Tailor the imported API to our needs...
+ */
+extend service API_BUSINESS_PARTNER with {
```

Comment:
This might be dangerous since our runtime assumes the extensions are included in the external service. It works well with mocks - but the real remote service is _immutable_. For this to work in our runtime we need to know which are the _original_ parts and which are the extensions, then we can make the translation.

Reply:
For projections this might be trivial to find out: Every projection is an extension (since remote service definitions don't have projections, right?).